apm-agent-plugins/apm-jmx-plugin/src/test/java/co/elastic/apm/agent/jmx/JmxMetricTrackerTest.java

```diff
@@ -83,6 +83,17 @@ void testGC() throws Exception {
         printMetricSets();
     }
 
+    @Test
+    void testAttributeWildcard() throws Exception {
+        setConfig(JmxMetric.valueOf("object_name[java.lang:type=GarbageCollector,name=*] attribute[*]"));
+        for (GarbageCollectorMXBean gcBean : ManagementFactory.getGarbageCollectorMXBeans()) {
+            String memoryManagerName = gcBean.getName();
+            assertThat(metricRegistry.getGaugeValue("jvm.jmx.collection_count", Labels.Mutable.of("name", memoryManagerName).add("type", "GarbageCollector"))).isNotNegative();
```

Comment:
This means that the internal JVM metrics are altered by the JMX configuration, which is not something I would have expected, but also probably helps to reproduce the issue that someone reported in our forum a few days ago, I'll investigate this further.

Reply:
I was too quick to conclude on this, it has the `jvm.jmx.` prefix here, thus while the metrics look similar to the default JVM metrics, those are not the same.